docs/JLBP-11.md

```diff
@@ -4,6 +4,18 @@ Release no later than 6 weeks after any of your dependencies
 releases a higher version. Time is important, not just how many versions behind,
 because a dependency tree that is up to date at each point in time is
 more likely to be internally consistent and compatible.
+Staying up to date is also important to ensure that security fixes are rolled
+out promptly.
+
+There are several tools that make upgrading dependencies easier by performing much of the grunt work:
+
+* The [Versions Maven Plugin](https://www.mojohaus.org/versions-maven-plugin/)
+  can inform you of new versions of dependencies and plugins or rewrite a
```

Comment:
For readability: "or rewrite" -> "or it can rewrite"

Reply:
rewrote